packages/core/react/src/WalletProvider.tsx

```diff
@@ -131,19 +131,23 @@ export const WalletProvider: FC<WalletProviderProps> = ({
     }, [isUnloading, setName]);
 
     // Handle the adapter's error event, and local errors
-    const onError = useCallback(
+    const handleError = useCallback(
```

Comment:
Rather than to differentiate the user-supplied handler from our synthesized one by an underscore, I left the user-supplied one as `onError` and renamed our synthesized one to the active-voice `handleError`.

Reply:
Changed the others for consistency.